test/e2e/security_context.go

```diff
@@ -95,6 +97,82 @@ var _ = framework.KubeDescribe("Security Context [Feature:SecurityContext]", fun
 		})
 	})
 
+	It("should support sysctls", func() {
```

Comment:
Can you move this test to under `test/e2e/common`? That way the test will be run under node e2e as well.